### PR TITLE
VET-80 Reminder Notification

### PR DESCRIPTION
**Pull request by Minhaajudeen**

### CODE REVIEW

File: app/screens/PetPrescription.js [hunk 1]
```diff
@@ -252,7 +251,7 @@ const PetPrescriptionScreen = ({ route }) => {
                       fontWeight: "700",
                     }}
                   >
-                    {pres.diagnosis}
```
Comment (AravindKumaran) [inline app/screens/PetPrescription.js:255]: it should be diagnosis. u got any conflicts?

File: app/screens/Room.js [hunk 2]
```diff
@@ -10,7 +10,6 @@ import {
   TouchableWithoutFeedback,
   Dimensions,
 } from "react-native";
-import KeepAwake from 'react-native-keep-awake';
```
Comment (AravindKumaran) [inline app/screens/Room.js:13]: why keep awake code is removed?

File: app/screens/Room.js [hunk 3]
```diff
@@ -221,29 +203,37 @@ const ChatRoom = ({ route }) => {
...
+    // }
   };
 
-  const sendPushToken = async (token, title, message, status = { notifyOnCallEnded: false }) => {
+  const sendPushToken = async (token, title, message) => {
     if (token) {
       setLoading(true);
 
-      let options = {
+      const pushRes = await usersApi.sendPushNotification({
```
Comment (AravindKumaran) [inline app/screens/Room.js:231]: call end notification code is also removed. why

File: app/screens/Room.js (hunk 2 above)
Comment (Minhaajudeen) [reply app/screens/Room.js:13]: that module is not used in that file

File: app/screens/HomeScreen.js [hunk 4]
```diff
@@ -621,7 +619,7 @@ const HomeScreen = ({ route }) => {
                       ))}
 
                     {
-                      vets.length <= 2
+                      vets.length <= 4
```
Comment (AravindKumaran) [inline app/screens/HomeScreen.js:622]: it should be 2 only.

**Pull request merged**